### PR TITLE
Changes all instances of 'coco' to 'cocoa'

### PR DESCRIPTION
**Pull request by TheVekter**
## About The Pull Request

Some denthead at some point wrote 'coco' instead of 'cocoa' and it's been bugging the hell out of me. I looked through the code, there is almost zero chance this breaks anything.

## Why It's Good For The Game

Because it's not _fucking spelled coco_

## Changelog
:cl:
spellcheck: Replaced all instances of 'coco' with 'cocoa'.
/:cl:

### CODE REVIEW

**Comment by Tumesuo** [comment]
cringe PR. delete

**Comment by spookydonut** [comment]
You missed some of the map templates.

**Comment by TheVekter** [comment]
> You missed some of the map templates.

Fixed~

**Comment by Fikou** [comment]
coco has SOUL

File: code/modules/food_and_drinks/food/snacks_pastry.dm [hunk 1]
```diff
@@ -111,7 +111,7 @@
 	name = "chocolate donut"
 	desc = "Goes great with a glass of warm milk."
 	icon_state = "donut_choc"
-	bonus_reagents = list(/datum/reagent/consumable/hot_coco = 3, /datum/reagent/consumable/sprinkles = 1) //the coco reagent is just bitter.
+	bonus_reagents = list(/datum/reagent/consumable/hot_coco = 3, /datum/reagent/consumable/sprinkles = 1) //the cocoa reagent is just bitter.
```
Comment (kriskog) [inline code/modules/food_and_drinks/food/snacks_pastry.dm]: Did you not change hot cocoa, but the comment?

File: code/modules/food_and_drinks/food/snacks_pastry.dm [hunk 2]
```diff
@@ -201,7 +201,7 @@
 	name = "chocolate jelly donut"
 	desc = "Goes great with a glass of warm milk."
 	icon_state = "jelly_choc"
-	bonus_reagents = list(/datum/reagent/consumable/hot_coco = 3, /datum/reagent/consumable/sprinkles = 1, /datum/reagent/consumable/nutriment/vitamin = 1) //the coco reagent is just bitter.
+	bonus_reagents = list(/datum/reagent/consumable/hot_coco = 3, /datum/reagent/consumable/sprinkles = 1, /datum/reagent/consumable/nutriment/vitamin = 1) //the cocoa reagent is just bitter.
```
Comment (kriskog) [inline code/modules/food_and_drinks/food/snacks_pastry.dm]: same

File: code/modules/food_and_drinks/food/snacks_pastry.dm [hunk 3]
```diff
@@ -282,7 +282,7 @@
...
```
Comment (kriskog) [inline code/modules/food_and_drinks/food/snacks_pastry.dm]: same

File: code/modules/reagents/chemistry/reagents/food_reagents.dm [hunk 4]
```diff
@@ -329,17 +329,17 @@
...
-	description = "A fatty, bitter paste made from coco beans."
+/datum/reagent/consumable/cocoa
+	name = "cocoa Powder"
+	description = "A fatty, bitter paste made from cocoa beans."
 	reagent_state = SOLID
 	nutriment_factor = 5 * REAGENTS_METABOLISM
 	color = "#302000" // rgb: 48, 32, 0
 	taste_description = "bitterness"
 
 /datum/reagent/consumable/hot_coco
```
Comment (kriskog) [inline code/modules/reagents/chemistry/reagents/food_reagents.dm]: actual reagent

**Review by kriskog — COMMENTED** [review]
You changed the text and comments but none of the paths for hot cocoa

File: code/modules/food_and_drinks/food/snacks_pastry.dm (hunk 1 above)
Comment (TheVekter) [reply code/modules/food_and_drinks/food/snacks_pastry.dm]: Really odd that these didn't change. I'll fix it shortly.

**Comment by itsadavid** [comment]
you stole my pr how dare you
https://github.com/tgstation/tgstation/pull/38442

**Comment by TheVekter** [comment]
Fixed it with that last push. Should be good now.

**Comment by Tlaltecuhtli** [comment]
do not remove the dutch hot coco its intended

**Comment by PKPenguin321** [comment]
What a dumb spelling
Cocoaaaaah

File: code/modules/food_and_drinks/recipes/food_mixtures.dm [hunk 5]
```diff
@@ -45,11 +45,11 @@
 		new /obj/item/reagent_containers/food/snacks/chocolatebar(location)
 	return
 
-/datum/chemical_reaction/hot_coco
-	name = "Hot Coco"
-	id = /datum/reagent/consumable/hot_coco
-	results = list(/datum/reagent/consumable/hot_coco = 5)
-	required_reagents = list(/datum/reagent/water = 5, /datum/reagent/consumable/coco = 1)
+/datum/chemical_reaction/hot_cocoa
+	name = "Hot cocoa"
```
Comment (ninjanomnom) [inline code/modules/food_and_drinks/recipes/food_mixtures.dm]: You flipped the case here
```suggestion
	name = "Hot Cocoa"
```

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

File: code/modules/reagents/chemistry/reagents/food_reagents.dm [hunk 6]
```diff
@@ -329,25 +329,25 @@
 	// no color (ie, black)
 	taste_description = "pepper"
 
-/datum/reagent/consumable/coco
-	name = "Coco Powder"
-	description = "A fatty, bitter paste made from coco beans."
+/datum/reagent/consumable/cocoa
+	name = "cocoa Powder"
```
Comment (ShizCalev) [inline code/modules/reagents/chemistry/reagents/food_reagents.dm]:
```suggestion
	name = "Cocoa Powder"
```

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/modules/food_and_drinks/drinks/drinks.dm [hunk 7]
```diff
@@ -242,10 +242,10 @@
 	desc = "An insult to Duke Purple is an insult to the Space Queen! Any proper gentleman will fight you, if you sully this tea."
 	list_reagents = list(/datum/reagent/consumable/tea = 30)
 
-/obj/item/reagent_containers/food/drinks/mug/coco
-	name = "Dutch hot coco"
+/obj/item/reagent_containers/food/drinks/mug/cocoa
+	name = "Dutch hot cocoa"
```
Comment (ShizCalev) [inline code/modules/food_and_drinks/drinks/drinks.dm]:
```suggestion
	name = "\improper Dutch hot cocoa"
```

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Comment by TheVekter** [comment]
There's a better way to handle this. I'll open a new one once I'm done.

**Pull request closed without merging**